src/ci/bin/test.inc.sh

```diff
@@ -114,6 +114,16 @@ cromwell::private::create_build_variables() {
         CROMWELL_BUILD_GIT_HASH_SUFFIX="gUNKNOWN"
     fi
 
+    if [[ "${TRAVIS_EVENT_TYPE}" == "pull_request" ]]; then
+        if git diff --name-only "${TRAVIS_BRANCH}" 2>&1 | grep -q --invert-match ^docs/; then
+            CROMWELL_BUILD_ONLY_DOCS_CHANGED=false
+        else
+            CROMWELL_BUILD_ONLY_DOCS_CHANGED=true
+        fi
+    else
+        CROMWELL_BUILD_ONLY_DOCS_CHANGED=false
+    fi
+
```

Comment:
TBH, I'm not sure why this is needed? Based on tracing through the logic below, this seems to short-circuit it rather than fix it (and now will `sbt` tests run even if docs were the only thing changed?)

Can we debug why the logic below is giving the wrong answer rather than adding a short-circuit-to-the-short-circuiting? Otherwise I'm a little worried it might also give the wrong answer in other situations.